Client/mods/deathmatch/logic/luadefs/CLuaDefs.h

```diff
@@ -64,4 +64,40 @@ class CLuaDefs
     static CClientDFFManager*         m_pDFFManager;
     static CClientColModelManager*    m_pColModelManager;
     static CRegisteredCommands*       m_pRegisteredCommands;
-};
\ No newline at end of file
+
+protected:
+    // Old style: Only warn on failure. This should
+    // not be used for new functions
+    template <auto T>
+    static inline int ArgumentParserWarn(lua_State* L)
+    {
+        return CLuaFunctionParser<false, T>()(L, m_pScriptDebugging);
+    }
+
+    // Special cases for overloads
+    template <auto T, auto U, auto... Ts>
```

Comment:
Even though `Special cases for overloads` is quite obvious, the fact that each of these template parameters are functions was not obvious to me.

Perhaps you could do
```suggestion
    template <auto F, auto G, auto... Fs>
```
or
```suggestion
    template <auto F, auto G, auto... Hs>
```
or even,
```suggestion
    template <auto F1, auto F2, auto... Fs>
```

I think the last one is the most obvious to me.